src/packages/solid/components/Metadata/Rating.tsx

```diff
@@ -47,7 +47,6 @@ const Rating: Component<RatingProps> = (props: RatingProps) => {
   const formattedTitle = createMemo(() => formatTitle(props.title));
   return (
     <View
-      {...props}
```

Comment:
if we rename `Rating.src` to `Rating.iconSrc` we can restore this props spread and the fix still works

Reply:
Updated, thanks